website/events-listener/package.json

```diff
@@ -1,23 +1,24 @@
 {
   "name": "@gear-js/event-listener",
-  "version": "0.1.1",
+  "version": "0.2.2",
   "description": "",
-  "main": "src/indes.js",
-  "type": "module",
+  "main": "src/indes.ts",
   "scripts": {
-    "start": "node dist/index.js",
-    "start:dev": "nodemon src/index.js",
-    "build": "cp -r src dist"
+    "start": "ts-node src/index.ts",
```

Comment:
I think it's better to use ts-node in `start:dev` script and add here `build` script with running `tsc` command and use `node build/index.js` in start script.